sources/sources.bib

```diff
@@ -25,3 +25,35 @@ @article{sample_ref
   publisher={Springer}
 }
 
+@INPROCEEDINGS{8308186,
+  author={S. {Albawi} and T. A. {Mohammed} and S. {Al-Zawi}},
+  booktitle={2017 International Conference on Engineering and Technology (ICET)}, 
+  title={Understanding of a convolutional neural network}, 
+  year={2017},
+  volume={},
```

Comment:
Empty `volume` and `number` attributes can be removed, I believe. Not sure why IEEE keeps exporting these attributes 🤔.